uiowa/uiowa
# Background color css

This is a start for #1654. This does not contain full work for people or events just quite yet, as those were unfinished at the time of creation. these could be revisited.

To Test:

1. pull down branch, composer install, blt frontend, drush cr
2. blt dsa the demo site from dev. you will need these pages to then test all of the items necessary:
    https://default.local.drupal.uiowa.edu/kitchen-sink
    https://default.local.drupal.uiowa.edu/kitchen-sink-webform
    https://default.local.drupal.uiowa.edu/kitchen-sink-carousel
3. check all variations and make sure they appear to be in a proper styled state.
